create.sh

```diff
@@ -116,10 +121,11 @@ sudo cp base/restart.sh ~atsign/atsign/etc/renewal-hooks/deploy
     tput setaf 2
 # It would be nice to use the @sign for the name but
 # Docker insists on a name that is DNS compliant and so emojis and @ signs are out hence the $SERVICE tag
-# we use a neat trick using docker-compose to create the compose file for us.
+# we use a neat trick usign docker-compose to create the compose file for us.
```

Comment:
This introduces a typo, it was previously correct.

Reply:
This must by typo on my side. I didnt intend to modify it